### PR TITLE
Wpc 20 duplicate articles and pages

### PR DESCRIPTION
**Pull request by HanaAyman**

### CODE REVIEW

**Review by 0815pascal — APPROVED** [review]

**Review by faebebin — APPROVED** [review]

**Review by bishoymelek — APPROVED** [review]
:tada: Great Job

File: packages/editor/src/client/routes/articleList.tsx [hunk 1]
```diff
@@ -256,7 +271,7 @@ export function ArticleList() {
         <Modal.Footer>
           <Button
             color={'red'}
-            disabled={isUnpublishing || isDeleting}
+            disabled={isUnpublishing || isDeleting || isDubplicating}
```
Comment (penk85) [inline packages/editor/src/client/routes/articleList.tsx:274]: isDuplicating

**Review by penk85 — APPROVED** [review]

File: packages/api/src/graphql/mutation.private.ts [hunk 2]
```diff
@@ -717,6 +717,23 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
...
+      type: GraphQLNonNull(GraphQLArticle),
+      args: {
+        id: {type: GraphQLNonNull(GraphQLID)}
+      },
+      async resolve(root, {id}, {dbAdapter, loaders}) {
+        const article = await loaders.articles.load(id)
+
+        if(!article) throw new NotFound('article', id)
+
+        const articleRevision = Object.assign(article.published ?? article.draft, { slug: ''}, { publishedAt: undefined }, { updatedAt: undefined })
```
Comment (roosnic1) [inline packages/api/src/graphql/mutation.private.ts:730]: you can do them in the same object. `{slug: '', publishedAt: undefined, updatedAt: undefined}`

File: packages/api/src/graphql/mutation.private.ts [hunk 3]
```diff
@@ -804,6 +821,23 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
...
+      type: GraphQLNonNull(GraphQLPage),
+      args: {
+        id: {type: GraphQLNonNull(GraphQLID)}
+      },
+      async resolve(root, {id}, {dbAdapter, loaders}) {
+        const page = await loaders.pages.load(id)
+
+        if(!page) throw new NotFound('page', id)
+
+        const pageRevision = Object.assign(page.published ?? page.draft, { slug: ''}, { publishedAt: undefined }, { updatedAt: undefined })
```
Comment (roosnic1) [inline packages/api/src/graphql/mutation.private.ts:834]: same here. you can do them in the same object

**Review by roosnic1 — COMMENTED** [review]
@HanaAyman can you do the changes in a new PR please?

**Pull request merged**